client/src/pages/ViewContestSubmissions/ViewContestSubmissions.tsx

```diff
@@ -0,0 +1,114 @@
+import react, { useState, useEffect } from 'react';
+import { Grid, Typography, Box, Chip, Avatar } from '@material-ui/core';
+import GridList from '@material-ui/core/GridList';
+import GridListTile from '@material-ui/core/GridListTile';
+import GridListTileBar from '@material-ui/core/GridListTileBar';
+import IconButton from '@material-ui/core/IconButton';
+import AddCircleOutlineIcon from '@material-ui/icons/AddCircleOutline';
+import useStyles from './useStyles';
+
+import { contest, contests } from '../../interface/tempContestData';
+// mock data for now
+const submissions = [
+  { img: 'https://source.unsplash.com/random', author: 'freddy', title: 'hello' },
+  { img: 'https://source.unsplash.com/1600x900/?nature,water', author: 'Roberta', title: 'Lion Tattoo' },
+  { img: 'https://source.unsplash.com/1600x900/?lions,tigers', author: 'Carlos', title: 'My Submission' },
+  { img: 'https://source.unsplash.com/1600x900/?tattoo,plants', author: 'Wendy', title: 'Pick Me!' },
+  { img: 'https://source.unsplash.com/1600x900/?plane,eyes', author: 'Frank', title: 'Fly away' },
+  { img: 'https://source.unsplash.com/1600x900/?comet,hello', author: 'jojoe', title: 'indiana jones' },
+  { img: 'https://source.unsplash.com/1600x900/?space,fire', author: 'miranda', title: 'contest time' },
+  { img: 'https://source.unsplash.com/1600x900/?grim,reaper', author: 'Sarah', title: 'you are my friend' },
+  { img: 'https://source.unsplash.com/1600x900/?brazil,country', author: 'Paul', title: 'what is up' },
+  { img: 'https://source.unsplash.com/1600x900/?forest,fight', author: 'Milad', title: 'hello' },
+  { img: 'https://source.unsplash.com/1600x900/?tank,fighter', author: 'Carlos', title: 'hello' },
+];
+
+export default function ViewContestSubmissions(): JSX.Element {
+  const classes = useStyles();
+  const [contestCard, setContestCard] = useState<contest>(Object);
+  const [winnerIndex, setWinnerIndex] = useState(Number);
```

Comment:
I think right now you are setting these default values to classes of Object and Number, not to an instance of them. Would be better to do something like this:
```suggestion
  const [contestCard, setContestCard] = useState<contest>(null);
  const [winnerIndex, setWinnerIndex] = useState(-1);
```
And in the case of the winner index, have some logic to check if it equals 0 or greater

Reply:
Was able to default winnerIndex to -1 but setContest card needed to be Object